core/tee/tee_svc_cryp.c

```diff
@@ -3207,13 +3226,20 @@ TEE_Result syscall_cipher_final(unsigned long state, const void *src,
 					    src, src_len, dst, dst_len);
 }
 
+/* Helper to factorize bounce buffers allocation in key derivation functions */
+struct bbuf_obj {
+	void *buf;
+	size_t len;
+};
+
 #if defined(CFG_CRYPTO_HKDF)
 static TEE_Result get_hkdf_params(uint32_t algo, const TEE_Attribute *params,
 				  uint32_t param_count,
 				  void **salt, size_t *salt_len, void **info,
 				  size_t *info_len, size_t *okm_len,
-				  uint32_t *hash_id)
+				  uint32_t *hash_id, struct bbuf_obj *bbufs)
```

Comment:
The output of `bbufs` is not used by the caller, it seems we can remove this parameter. Same for the other `get_*_params()` below.